Regression on cross compiling to aarch64-unknown-linux-musl using nightly

### Code

I do not have minimised code. This was a sudden CI build failure on nightly for aarch64-unknown-linux-musl ([ci logs](https://github.com/VorpalBlade/paketkoll/actions/runs/10164190374/job/28108962786#logs))

Note that this is a cross compile using cross-rs.

It fails on rust version 1.82.0-nightly (612a33f20 2024-07-29) but works on 1.82.0-nightly (2cbbe8b8b 2024-07-28). Needless to say it also works perfectly fine on stable.

* The error is reproducible on rebuild and locally with cross-rs.
* It is not reproducible with cargo-zigbuild interestingly enough.
* I also checked that the version of cross-rs didn't change between the successful and failing builds (no new releases of that project since february).
* The only change between the successful and failing CI runs were a typo fix to an unrelated CI pipeline (the release one, so it didn't even run yet).

As such I'm forced to conclude that the blame lies with Rustc.

Steps to reproduce:

```bash
git clone https://github.com/VorpalBlade/paketkoll
cd paketkoll
git checkout 48867a025c0f2192317b7e75ddaf75fce7da3f58
cross +nightly build --target aarch64-unknown-linux-musl
```

I expected to see this happen: Compilation successful (as it is with stable for same target, or with -gnu and nightly)

Instead, this happened ([full ci logs](https://github.com/VorpalBlade/paketkoll/actions/runs/10164190374/job/28108962786#logs))

```
      Running `rustc --crate-name rune --edition=2021 /cargo/registry/src/index.crates.io-6f17d22bba15001f/rune-0.13.4/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts,future-incompat --crate-type lib --emit=dep-info,metadata,link -C embed-bitcode=no -C debuginfo=1 -C split-debuginfo=packed --cfg 'feature="alloc"' --cfg 'feature="anyhow"' --cfg 'feature="base64"' --cfg 'feature="bincode"' --cfg 'feature="capture-io"' --cfg 'feature="clap"' --cfg 'feature="cli"' --cfg 'feature="codespan-reporting"' --cfg 'feature="default"' --cfg 'feature="disable-io"' --cfg 'feature="doc"' --cfg 'feature="emit"' --cfg 'feature="fmt"' --cfg 'feature="handlebars"' --cfg 'feature="languageserver"' --cfg 'feature="linked-hash-map"' --cfg 'feature="lsp"' --cfg 'feature="musli-storage"' --cfg 'feature="parking_lot"' --cfg 'feature="percent-encoding"' --cfg 'feature="pulldown-cmark"' --cfg 'feature="rand"' --cfg 'feature="relative-path"' --cfg 'feature="ropey"' --cfg 'feature="rust-embed"' --cfg 'feature="semver"' --cfg 'feature="serde-hashkey"' --cfg 'feature="serde_json"' --cfg 'feature="sha2"' --cfg 'feature="similar"' --cfg 'feature="std"' --cfg 'feature="syntect"' --cfg 'feature="tokio"' --cfg 'feature="toml"' --cfg 'feature="tracing-subscriber"' --cfg 'feature="url"' --cfg 'feature="webbrowser"' --cfg 'feature="workspace"' --check-cfg 'cfg(docsrs)' --check-cfg 'cfg(feature, values("alloc", "anyhow", "base64", "bench", "bincode", "byte-code", "capture-io", "clap", "cli", "codespan-reporting", "default", "disable-io", "doc", "emit", "fmt", "handlebars", "languageserver", "linked-hash-map", "lsp", "musli-storage", "parking_lot", "percent-encoding", "pulldown-cmark", "rand", "relative-path", "ropey", "rust-embed", "semver", "serde-hashkey", "serde_json", "sha2", "similar", "std", "syntect", "tokio", "toml", "tracing-subscriber", "url", "webbrowser", "workspace"))' -C metadata=b7e3f8fc1bcdb225 -C extra-filename=-b7e3f8fc1bcdb225 --out-dir /target/aarch64-unknown-linux-musl/debug/deps --target aarch64-unknown-linux-musl -C linker=aarch64-linux-musl-gcc.sh -L dependency=/target/aarch64-unknown-linux-musl/debug/deps -L dependency=/target/debug/deps --extern anyhow=/target/aarch64-unknown-linux-musl/debug/deps/libanyhow-ace75f78c86bb2bf.rmeta --extern base64=/target/aarch64-unknown-linux-musl/debug/deps/libbase64-4afbf02b7a7e7a3a.rmeta --extern bincode=/target/aarch64-unknown-linux-musl/debug/deps/libbincode-bf5632fd9e007c02.rmeta --extern clap=/target/aarch64-unknown-linux-musl/debug/deps/libclap-7366d26291cb12fd.rmeta --extern codespan_reporting=/target/aarch64-unknown-linux-musl/debug/deps/libcodespan_reporting-384338113a5d0c48.rmeta --extern futures_core=/target/aarch64-unknown-linux-musl/debug/deps/libfutures_core-4b460c0fe95d39bf.rmeta --extern futures_util=/target/aarch64-unknown-linux-musl/debug/deps/libfutures_util-392b494464649bd9.rmeta --extern handlebars=/target/aarch64-unknown-linux-musl/debug/deps/libhandlebars-f18acd27d5a85a5d.rmeta --extern itoa=/target/aarch64-unknown-linux-musl/debug/deps/libitoa-1e607d164f4083d3.rmeta --extern linked_hash_map=/target/aarch64-unknown-linux-musl/debug/deps/liblinked_hash_map-b34c2b350eb6cb06.rmeta --extern lsp=/target/aarch64-unknown-linux-musl/debug/deps/liblsp_types-18a4bc0a07ab1fea.rmeta --extern musli=/target/aarch64-unknown-linux-musl/debug/deps/libmusli-2b264c835241d851.rmeta --extern musli_storage=/target/aarch64-unknown-linux-musl/debug/deps/libmusli_storage-394033d2ccd84d49.rmeta --extern num=/target/aarch64-unknown-linux-musl/debug/deps/libnum-12f585f05aaa4840.rmeta --extern once_cell=/target/aarch64-unknown-linux-musl/debug/deps/libonce_cell-85a46c04a094204c.rmeta --extern parking_lot=/target/aarch64-unknown-linux-musl/debug/deps/libparking_lot-7cf960452f5a6b17.rmeta --extern percent_encoding=/target/aarch64-unknown-linux-musl/debug/deps/libpercent_encoding-7d506d6cb45e6973.rmeta --extern pin_project=/target/aarch64-unknown-linux-musl/debug/deps/libpin_project-6ef83f43b59050d4.rmeta --extern pulldown_cmark=/target/aarch64-unknown-linux-musl/debug/deps/libpulldown_cmark-74c165cd3e996a3a.rmeta --extern rand=/target/aarch64-unknown-linux-musl/debug/deps/librand-e2408752c9cf6076.rmeta --extern relative_path=/target/aarch64-unknown-linux-musl/debug/deps/librelative_path-9a53023830e9a786.rmeta --extern ropey=/target/aarch64-unknown-linux-musl/debug/deps/libropey-9fa6ebbce3ea12c1.rmeta --extern rune_alloc=/target/aarch64-unknown-linux-musl/debug/deps/librune_alloc-9a23e4e60c221971.rmeta --extern rune_core=/target/aarch64-unknown-linux-musl/debug/deps/librune_core-f813eb532724d6a4.rmeta --extern rune_macros=/target/debug/deps/librune_macros-ffee5c0c8f224a6f.so --extern rust_embed=/target/aarch64-unknown-linux-musl/debug/deps/librust_embed-381e2197424e4975.rmeta --extern ryu=/target/aarch64-unknown-linux-musl/debug/deps/libryu-e8bca8abb3d31978.rmeta --extern semver=/target/aarch64-unknown-linux-musl/debug/deps/libsemver-9e120f9117b5821a.rmeta --extern serde=/target/aarch64-unknown-linux-musl/debug/deps/libserde-4be2fe20c57f2b9e.rmeta --extern serde_hashkey=/target/aarch64-unknown-linux-musl/debug/deps/libserde_hashkey-15443b005d996941.rmeta --extern serde_json=/target/aarch64-unknown-linux-musl/debug/deps/libserde_json-c0bfa577697a09b1.rmeta --extern sha2=/target/aarch64-unknown-linux-musl/debug/deps/libsha2-5048850b029849ef.rmeta --extern similar=/target/aarch64-unknown-linux-musl/debug/deps/libsimilar-4a02cafb33168dc1.rmeta --extern syntect=/target/aarch64-unknown-linux-musl/debug/deps/libsyntect-24a886f1fdf6f845.rmeta --extern tokio=/target/aarch64-unknown-linux-musl/debug/deps/libtokio-1452aa1ab50a9ca6.rmeta --extern toml=/target/aarch64-unknown-linux-musl/debug/deps/libtoml-a3add7273c8bc7bb.rmeta --extern tracing=/target/aarch64-unknown-linux-musl/debug/deps/libtracing-78fe2a9d80ce56ef.rmeta --extern tracing_subscriber=/target/aarch64-unknown-linux-musl/debug/deps/libtracing_subscriber-80ae2a31d0a26d89.rmeta --extern url=/target/aarch64-unknown-linux-musl/debug/deps/liburl-328bbe7e2773d98e.rmeta --extern webbrowser=/target/aarch64-unknown-linux-musl/debug/deps/libwebbrowser-e44633b8cea71b10.rmeta --cap-lints allow -D warnings -Clink-arg=-Wl,--compress-debug-sections=zlib -L native=/target/aarch64-unknown-linux-musl/debug/build/libz-ng-sys-dfaa21d16411c9b8/out/lib -L native=/target/aarch64-unknown-linux-musl/debug/build/libz-ng-sys-dfaa21d16411c9b8/out/lib64 -L native=/target/aarch64-unknown-linux-musl/debug/build/onig_sys-f637ddd37320f1c0/out`
error: linking with `aarch64-linux-musl-gcc.sh` failed: exit status: 1
  |
  = note: LC_ALL="C" PATH="/rust/lib/rustlib/x86_64-unknown-linux-gnu/bin:/rust/lib/rustlib/x86_64-unknown-linux-gnu/bin/self-contained:/rust/lib/rustlib/x86_64-unknown-linux-gnu/bin:/rust/lib/rustlib/x86_64-unknown-linux-gnu/bin/self-contained:/rust/lib/rustlib/x86_64-unknown-linux-gnu/bin:/rust/lib/rustlib/x86_64-unknown-linux-gnu/bin/self-contained:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/rust/bin" VSLANG="1033" "aarch64-linux-musl-gcc.sh" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crt1.o" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crti.o" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crtbegin.o" "/tmp/rustc7zZeCI/symbols.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.00.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.01.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.02.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.03.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.04.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.05.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.06.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.07.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.08.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.09.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.10.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.11.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.12.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.13.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.14.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.15.rcgu.o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.a3svcrwtoayw76rvpk50cyxul.rcgu.o" "-Wl,--as-needed" "-L" "/target/aarch64-unknown-linux-musl/debug/deps" "-L" "/target/debug/deps" "-L" "/target/aarch64-unknown-linux-musl/debug/build/libmimalloc-sys-3ba65218e0db3490/out" "-L" "/target/aarch64-unknown-linux-musl/debug/build/bzip2-sys-7abe98856e374977/out/lib" "-L" "/target/aarch64-unknown-linux-musl/debug/build/libz-ng-sys-dfaa21d16411c9b8/out/lib" "-L" "/target/aarch64-unknown-linux-musl/debug/build/libz-ng-sys-dfaa21d16411c9b8/out/lib64" "-L" "/target/aarch64-unknown-linux-musl/debug/build/ring-0fa7d8a90b68f2b3/out" "-L" "/target/aarch64-unknown-linux-musl/debug/build/lzma-sys-ffe061071f20c9d7/out" "-L" "/target/aarch64-unknown-linux-musl/debug/build/zstd-sys-8ec35f6b42db7305/out" "-L" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib" "-Wl,-Bstatic" "/target/aarch64-unknown-linux-musl/debug/deps/libserde_json-c0bfa577697a09b1.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libenv_logger-374e52d9cbc437b8.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libenv_filter-84fd749340938abf.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libhumantime-0ffb54a9b9a9b792.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libpaketkoll-ea54951be33e3d54.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libos_info-98fc2186f1d54fb8.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libpaketkoll_core-ae547def72c3c81d.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libglob-1ffa279201be60cf.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libbzip2-945f3f3f226a53e1.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libbzip2_sys-f7690df051249ee2.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libxz2-cd3cb3c69fb8d444.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/liblzma_sys-7ba82b51a8331d3d.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libflume-fdb1a33c515be94a.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libnum_cpus-f2b76db177a41b30.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libar-163ea80ebbacf220.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libtar-04148244a17ea86e.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libxattr-3f9ae327ef84a2da.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/librustix-b0b9d34f3ef1e78f.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/liblinux_raw_sys-21b20448246edf68.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libfiletime-64bc801f6c017776.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libzstd-aec18906597eb4f0.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libzstd_safe-e2a63584664c1791.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libzstd_sys-32669feaf9b0441b.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libparking_lot-7cf960452f5a6b17.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libini-5696f8e086681921.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libtrim_in_place-15ef2ebe75ec2ccb.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libordered_multimap-f2cafd2bda2d6c6c.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libdlv_list-a95616a16286057c.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libphf-37f30b4232bccf14.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libphf_shared-3a4680b25332d1d8.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libsiphasher-d8e0f84d169cc7fc.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libignore-96c9f1a3677e5739.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libwalkdir-999bce26bf374208.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libsame_file-358a8f2db3e80589.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libglobset-c3c2301d06d0a237.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/liblog-2e501f34727b15ac.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libmd5-1b031c122055d5bd.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libdigest-eb9701d75aedabd9.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libblock_buffer-f959b7b325b2998d.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libcrypto_common-136002bbd9c8e538.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libgeneric_array-fce974acd905d371.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libtypenum-982618236dc3952f.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libpaketkoll_utils-c0238f7f4ee8aa17.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libring-54baa33c753abe23.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libspin-95720c1a8683caf1.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libuntrusted-2c9fa63b699cec54.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libmtree2-1df96acda0a011a5.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libflate2-e57f32eeb579768a.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/liblibz_ng_sys-2df53b732c0ea07a.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libcrc32fast-fa30ed19936b0fb9.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libregex-b1c38795bf4b10a6.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libbstr-885d1ea510e380d0.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libregex_automata-645483a204ad645a.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libaho_corasick-5447e2cf3fdc87dc.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libregex_syntax-49fe88d2665ab5bd.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libmemchr-2c1d667111a5d069.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libpaketkoll_types-07c7b66d6d55f314.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/liblasso-f5da76b4a5991cb2.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libdashmap-2a3b7ef5de893ca4.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libhashbrown-1218c9c558352c87.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libfaster_hex-a61728ceb202e328.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libserde_bytes-802fb2cbb58fefab.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libnix-394b1843d91b4cbb.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libderive_builder-ce8f3714191c2764.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libbitflags-fd4f6c46ab290a5f.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libthiserror-ca4c9ada60adf778.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libstrum-178ce37859f88706.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libdashmap-aee948a18804644c.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/liblock_api-f585f6dc36ad1b57.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libscopeguard-8bd0d056ec1684c0.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libhashbrown-37e22e4fc15549f8.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/liballocator_api2-b679d84ff60160ae.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libparking_lot_core-a7181e9afaae8341.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libcompact_str-6987da3bf8766e73.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libryu-e8bca8abb3d31978.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libitoa-1e607d164f4083d3.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libstatic_assertions-fc2c5dfa1d1a5dab.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libcastaway-19be1111bd3c659f.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libsmallvec-7c37c71e4d75c193.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libserde-4be2fe20c57f2b9e.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libmimalloc-b7738520f972514e.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/liblibmimalloc_sys-6c0e9186df988277.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/librayon-39c1a8c4f41afd61.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/librayon_core-411c37910918dfbe.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libcrossbeam_deque-958d910d3fc9c7f0.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libcrossbeam_epoch-951dda309020f21e.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libcrossbeam_utils-cc5589b8e8f15bbc.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libeither-4a3ae4cfdcf7bdb8.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libproc_exit-8498b1338578edba.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libclap-7366d26291cb12fd.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libclap_builder-c4c5e5dcb350e76a.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libstrsim-06b7fead5a98fecb.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libanstream-67872ffc57a29db7.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libanstyle_query-7780f943e8b484c0.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libis_terminal_polyfill-3c6fb2c6797f50c5.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libcolorchoice-c94897b0d0c571bb.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libanstyle_parse-c0dc749435b4e727.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libutf8parse-bb165e27c62f6a5d.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libclap_lex-02eb4a5e56d4b271.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libanstyle-0c22d89388f1b183.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libanyhow-ace75f78c86bb2bf.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libahash-a13908ed97ec726c.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libgetrandom-1a57b3a48ba86154.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/liblibc-57bc4bd4acb02d83.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libonce_cell-85a46c04a094204c.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libportable_atomic-89d5fc52adbbe488.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libcfg_if-dd139f6561fa45b0.rlib" "/target/aarch64-unknown-linux-musl/debug/deps/libzerocopy-dc998ed92317929b.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libstd-6e22b5d64cda649f.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libpanic_unwind-6dbdda2b1896f9f5.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libobject-7c15364b571acde3.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libmemchr-5f08222295da918c.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libaddr2line-b3271a0e792541e1.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libgimli-ddb92b1d3a7f3fa9.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/librustc_demangle-0f203aa5e4445b63.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libstd_detect-51bc41b0313f22d5.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libhashbrown-c6e9d546aea92afd.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/librustc_std_workspace_alloc-6a0e538de4ea440e.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libminiz_oxide-d08061ebdb7684ff.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libadler-3c4c44baabf4dded.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libunwind-b9f2df18bad75795.rlib" "-lunwind" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libcfg_if-3d127a20420d1d03.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/liblibc-f1d0ee652437bf5c.rlib" "-lc" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/liballoc-a2d865e3d00a5fb6.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/librustc_std_workspace_core-eb3ad2970efcad72.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libcore-2a67b8b32bd9e6f3.rlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libcompiler_builtins-355b08253bc8d828.rlib" "-Wl,-Bdynamic" "-Wl,--eh-frame-hdr" "-Wl,-z,noexecstack" "-nostartfiles" "-L" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib" "-L" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained" "-o" "/target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20" "-Wl,--gc-sections" "-static" "-no-pie" "-Wl,-z,relro,-z,now" "-nodefaultlibs" "-Wl,--compress-debug-sections=zlib" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crtend.o" "/rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crtn.o"
  = note: + set -euo pipefail
          + main /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crt1.o /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crti.o /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crtbegin.o /tmp/rustc7zZeCI/symbols.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.00.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.01.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.02.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.03.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.04.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.05.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.06.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.07.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.08.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.09.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.10.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.11.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.12.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.13.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.14.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.15.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.a3svcrwtoayw76rvpk50cyxul.rcgu.o -Wl,--as-needed -L /target/aarch64-unknown-linux-musl/debug/deps -L /target/debug/deps -L /target/aarch64-unknown-linux-musl/debug/build/libmimalloc-sys-3ba65218e0db3490/out -L /target/aarch64-unknown-linux-musl/debug/build/bzip2-sys-7abe98856e374977/out/lib -L /target/aarch64-unknown-linux-musl/debug/build/libz-ng-sys-dfaa21d16411c9b8/out/lib -L /target/aarch64-unknown-linux-musl/debug/build/libz-ng-sys-dfaa21d16411c9b8/out/lib64 -L /target/aarch64-unknown-linux-musl/debug/build/ring-0fa7d8a90b68f2b3/out -L /target/aarch64-unknown-linux-musl/debug/build/lzma-sys-ffe061071f20c9d7/out -L /target/aarch64-unknown-linux-musl/debug/build/zstd-sys-8ec35f6b42db7305/out -L /rust/lib/rustlib/aarch64-unknown-linux-musl/lib -Wl,-Bstatic /target/aarch64-unknown-linux-musl/debug/deps/libserde_json-c0bfa577697a09b1.rlib /target/aarch64-unknown-linux-musl/debug/deps/libenv_logger-374e52d9cbc437b8.rlib /target/aarch64-unknown-linux-musl/debug/deps/libenv_filter-84fd749340938abf.rlib /target/aarch64-unknown-linux-musl/debug/deps/libhumantime-0ffb54a9b9a9b792.rlib /target/aarch64-unknown-linux-musl/debug/deps/libpaketkoll-ea54951be33e3d54.rlib /target/aarch64-unknown-linux-musl/debug/deps/libos_info-98fc2186f1d54fb8.rlib /target/aarch64-unknown-linux-musl/debug/deps/libpaketkoll_core-ae547def72c3c81d.rlib /target/aarch64-unknown-linux-musl/debug/deps/libglob-1ffa279201be60cf.rlib /target/aarch64-unknown-linux-musl/debug/deps/libbzip2-945f3f3f226a53e1.rlib /target/aarch64-unknown-linux-musl/debug/deps/libbzip2_sys-f7690df051249ee2.rlib /target/aarch64-unknown-linux-musl/debug/deps/libxz2-cd3cb3c69fb8d444.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblzma_sys-7ba82b51a8331d3d.rlib /target/aarch64-unknown-linux-musl/debug/deps/libflume-fdb1a33c515be94a.rlib /target/aarch64-unknown-linux-musl/debug/deps/libnum_cpus-f2b76db177a41b30.rlib /target/aarch64-unknown-linux-musl/debug/deps/libar-163ea80ebbacf220.rlib /target/aarch64-unknown-linux-musl/debug/deps/libtar-04148244a17ea86e.rlib /target/aarch64-unknown-linux-musl/debug/deps/libxattr-3f9ae327ef84a2da.rlib /target/aarch64-unknown-linux-musl/debug/deps/librustix-b0b9d34f3ef1e78f.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblinux_raw_sys-21b20448246edf68.rlib /target/aarch64-unknown-linux-musl/debug/deps/libfiletime-64bc801f6c017776.rlib /target/aarch64-unknown-linux-musl/debug/deps/libzstd-aec18906597eb4f0.rlib /target/aarch64-unknown-linux-musl/debug/deps/libzstd_safe-e2a63584664c1791.rlib /target/aarch64-unknown-linux-musl/debug/deps/libzstd_sys-32669feaf9b0441b.rlib /target/aarch64-unknown-linux-musl/debug/deps/libparking_lot-7cf960452f5a6b17.rlib /target/aarch64-unknown-linux-musl/debug/deps/libini-5696f8e086681921.rlib /target/aarch64-unknown-linux-musl/debug/deps/libtrim_in_place-15ef2ebe75ec2ccb.rlib /target/aarch64-unknown-linux-musl/debug/deps/libordered_multimap-f2cafd2bda2d6c6c.rlib /target/aarch64-unknown-linux-musl/debug/deps/libdlv_list-a95616a16286057c.rlib /target/aarch64-unknown-linux-musl/debug/deps/libphf-37f30b4232bccf14.rlib /target/aarch64-unknown-linux-musl/debug/deps/libphf_shared-3a4680b25332d1d8.rlib /target/aarch64-unknown-linux-musl/debug/deps/libsiphasher-d8e0f84d169cc7fc.rlib /target/aarch64-unknown-linux-musl/debug/deps/libignore-96c9f1a3677e5739.rlib /target/aarch64-unknown-linux-musl/debug/deps/libwalkdir-999bce26bf374208.rlib /target/aarch64-unknown-linux-musl/debug/deps/libsame_file-358a8f2db3e80589.rlib /target/aarch64-unknown-linux-musl/debug/deps/libglobset-c3c2301d06d0a237.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblog-2e501f34727b15ac.rlib /target/aarch64-unknown-linux-musl/debug/deps/libmd5-1b031c122055d5bd.rlib /target/aarch64-unknown-linux-musl/debug/deps/libdigest-eb9701d75aedabd9.rlib /target/aarch64-unknown-linux-musl/debug/deps/libblock_buffer-f959b7b325b2998d.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcrypto_common-136002bbd9c8e538.rlib /target/aarch64-unknown-linux-musl/debug/deps/libgeneric_array-fce974acd905d371.rlib /target/aarch64-unknown-linux-musl/debug/deps/libtypenum-982618236dc3952f.rlib /target/aarch64-unknown-linux-musl/debug/deps/libpaketkoll_utils-c0238f7f4ee8aa17.rlib /target/aarch64-unknown-linux-musl/debug/deps/libring-54baa33c753abe23.rlib /target/aarch64-unknown-linux-musl/debug/deps/libspin-95720c1a8683caf1.rlib /target/aarch64-unknown-linux-musl/debug/deps/libuntrusted-2c9fa63b699cec54.rlib /target/aarch64-unknown-linux-musl/debug/deps/libmtree2-1df96acda0a011a5.rlib /target/aarch64-unknown-linux-musl/debug/deps/libflate2-e57f32eeb579768a.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblibz_ng_sys-2df53b732c0ea07a.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcrc32fast-fa30ed19936b0fb9.rlib /target/aarch64-unknown-linux-musl/debug/deps/libregex-b1c38795bf4b10a6.rlib /target/aarch64-unknown-linux-musl/debug/deps/libbstr-885d1ea510e380d0.rlib /target/aarch64-unknown-linux-musl/debug/deps/libregex_automata-645483a204ad645a.rlib /target/aarch64-unknown-linux-musl/debug/deps/libaho_corasick-5447e2cf3fdc87dc.rlib /target/aarch64-unknown-linux-musl/debug/deps/libregex_syntax-49fe88d2665ab5bd.rlib /target/aarch64-unknown-linux-musl/debug/deps/libmemchr-2c1d667111a5d069.rlib /target/aarch64-unknown-linux-musl/debug/deps/libpaketkoll_types-07c7b66d6d55f314.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblasso-f5da76b4a5991cb2.rlib /target/aarch64-unknown-linux-musl/debug/deps/libdashmap-2a3b7ef5de893ca4.rlib /target/aarch64-unknown-linux-musl/debug/deps/libhashbrown-1218c9c558352c87.rlib /target/aarch64-unknown-linux-musl/debug/deps/libfaster_hex-a61728ceb202e328.rlib /target/aarch64-unknown-linux-musl/debug/deps/libserde_bytes-802fb2cbb58fefab.rlib /target/aarch64-unknown-linux-musl/debug/deps/libnix-394b1843d91b4cbb.rlib /target/aarch64-unknown-linux-musl/debug/deps/libderive_builder-ce8f3714191c2764.rlib /target/aarch64-unknown-linux-musl/debug/deps/libbitflags-fd4f6c46ab290a5f.rlib /target/aarch64-unknown-linux-musl/debug/deps/libthiserror-ca4c9ada60adf778.rlib /target/aarch64-unknown-linux-musl/debug/deps/libstrum-178ce37859f88706.rlib /target/aarch64-unknown-linux-musl/debug/deps/libdashmap-aee948a18804644c.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblock_api-f585f6dc36ad1b57.rlib /target/aarch64-unknown-linux-musl/debug/deps/libscopeguard-8bd0d056ec1684c0.rlib /target/aarch64-unknown-linux-musl/debug/deps/libhashbrown-37e22e4fc15549f8.rlib /target/aarch64-unknown-linux-musl/debug/deps/liballocator_api2-b679d84ff60160ae.rlib /target/aarch64-unknown-linux-musl/debug/deps/libparking_lot_core-a7181e9afaae8341.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcompact_str-6987da3bf8766e73.rlib /target/aarch64-unknown-linux-musl/debug/deps/libryu-e8bca8abb3d31978.rlib /target/aarch64-unknown-linux-musl/debug/deps/libitoa-1e607d164f4083d3.rlib /target/aarch64-unknown-linux-musl/debug/deps/libstatic_assertions-fc2c5dfa1d1a5dab.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcastaway-19be1111bd3c659f.rlib /target/aarch64-unknown-linux-musl/debug/deps/libsmallvec-7c37c71e4d75c193.rlib /target/aarch64-unknown-linux-musl/debug/deps/libserde-4be2fe20c57f2b9e.rlib /target/aarch64-unknown-linux-musl/debug/deps/libmimalloc-b7738520f972514e.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblibmimalloc_sys-6c0e9186df988277.rlib /target/aarch64-unknown-linux-musl/debug/deps/librayon-39c1a8c4f41afd61.rlib /target/aarch64-unknown-linux-musl/debug/deps/librayon_core-411c37910918dfbe.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcrossbeam_deque-958d910d3fc9c7f0.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcrossbeam_epoch-951dda309020f21e.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcrossbeam_utils-cc5589b8e8f15bbc.rlib /target/aarch64-unknown-linux-musl/debug/deps/libeither-4a3ae4cfdcf7bdb8.rlib /target/aarch64-unknown-linux-musl/debug/deps/libproc_exit-8498b1338578edba.rlib /target/aarch64-unknown-linux-musl/debug/deps/libclap-7366d26291cb12fd.rlib /target/aarch64-unknown-linux-musl/debug/deps/libclap_builder-c4c5e5dcb350e76a.rlib /target/aarch64-unknown-linux-musl/debug/deps/libstrsim-06b7fead5a98fecb.rlib /target/aarch64-unknown-linux-musl/debug/deps/libanstream-67872ffc57a29db7.rlib /target/aarch64-unknown-linux-musl/debug/deps/libanstyle_query-7780f943e8b484c0.rlib /target/aarch64-unknown-linux-musl/debug/deps/libis_terminal_polyfill-3c6fb2c6797f50c5.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcolorchoice-c94897b0d0c571bb.rlib /target/aarch64-unknown-linux-musl/debug/deps/libanstyle_parse-c0dc749435b4e727.rlib /target/aarch64-unknown-linux-musl/debug/deps/libutf8parse-bb165e27c62f6a5d.rlib /target/aarch64-unknown-linux-musl/debug/deps/libclap_lex-02eb4a5e56d4b271.rlib /target/aarch64-unknown-linux-musl/debug/deps/libanstyle-0c22d89388f1b183.rlib /target/aarch64-unknown-linux-musl/debug/deps/libanyhow-ace75f78c86bb2bf.rlib /target/aarch64-unknown-linux-musl/debug/deps/libahash-a13908ed97ec726c.rlib /target/aarch64-unknown-linux-musl/debug/deps/libgetrandom-1a57b3a48ba86154.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblibc-57bc4bd4acb02d83.rlib /target/aarch64-unknown-linux-musl/debug/deps/libonce_cell-85a46c04a094204c.rlib /target/aarch64-unknown-linux-musl/debug/deps/libportable_atomic-89d5fc52adbbe488.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcfg_if-dd139f6561fa45b0.rlib /target/aarch64-unknown-linux-musl/debug/deps/libzerocopy-dc998ed92317929b.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libstd-6e22b5d64cda649f.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libpanic_unwind-6dbdda2b1896f9f5.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libobject-7c15364b571acde3.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libmemchr-5f08222295da918c.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libaddr2line-b3271a0e792541e1.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libgimli-ddb92b1d3a7f3fa9.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/librustc_demangle-0f203aa5e4445b63.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libstd_detect-51bc41b0313f22d5.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libhashbrown-c6e9d546aea92afd.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/librustc_std_workspace_alloc-6a0e538de4ea440e.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libminiz_oxide-d08061ebdb7684ff.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libadler-3c4c44baabf4dded.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libunwind-b9f2df18bad75795.rlib -lunwind /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libcfg_if-3d127a20420d1d03.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/liblibc-f1d0ee652437bf5c.rlib -lc /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/liballoc-a2d865e3d00a5fb6.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/librustc_std_workspace_core-eb3ad2970efcad72.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libcore-2a67b8b32bd9e6f3.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libcompiler_builtins-355b08253bc8d828.rlib -Wl,-Bdynamic -Wl,--eh-frame-hdr -Wl,-z,noexecstack -nostartfiles -L /rust/lib/rustlib/aarch64-unknown-linux-musl/lib -L /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained -o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20 -Wl,--gc-sections -static -no-pie -Wl,-z,relro,-z,now -nodefaultlibs -Wl,--compress-debug-sections=zlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crtend.o /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crtn.o
          + local release=
          ++ rustc -Vv
          ++ grep '^release:'
          ++ cut -d : -f2
          + release=' 1.82.0-nightly'
          + local minor=
          ++ cut -d . -f2
          ++ echo ' 1.82.0-nightly'
          + minor=82
          + ((  minor >= 48  ))
          + aarch64-linux-musl-gcc /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crt1.o /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crti.o /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crtbegin.o /tmp/rustc7zZeCI/symbols.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.00.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.01.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.02.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.03.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.04.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.05.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.06.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.07.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.08.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.09.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.10.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.11.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.12.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.13.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.14.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.paketkoll.3881c0ba99d34f0-cgu.15.rcgu.o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20.a3svcrwtoayw76rvpk50cyxul.rcgu.o -Wl,--as-needed -L /target/aarch64-unknown-linux-musl/debug/deps -L /target/debug/deps -L /target/aarch64-unknown-linux-musl/debug/build/libmimalloc-sys-3ba65218e0db3490/out -L /target/aarch64-unknown-linux-musl/debug/build/bzip2-sys-7abe98856e374977/out/lib -L /target/aarch64-unknown-linux-musl/debug/build/libz-ng-sys-dfaa21d16411c9b8/out/lib -L /target/aarch64-unknown-linux-musl/debug/build/libz-ng-sys-dfaa21d16411c9b8/out/lib64 -L /target/aarch64-unknown-linux-musl/debug/build/ring-0fa7d8a90b68f2b3/out -L /target/aarch64-unknown-linux-musl/debug/build/lzma-sys-ffe061071f20c9d7/out -L /target/aarch64-unknown-linux-musl/debug/build/zstd-sys-8ec35f6b42db7305/out -L /rust/lib/rustlib/aarch64-unknown-linux-musl/lib -Wl,-Bstatic /target/aarch64-unknown-linux-musl/debug/deps/libserde_json-c0bfa577697a09b1.rlib /target/aarch64-unknown-linux-musl/debug/deps/libenv_logger-374e52d9cbc437b8.rlib /target/aarch64-unknown-linux-musl/debug/deps/libenv_filter-84fd749340938abf.rlib /target/aarch64-unknown-linux-musl/debug/deps/libhumantime-0ffb54a9b9a9b792.rlib /target/aarch64-unknown-linux-musl/debug/deps/libpaketkoll-ea54951be33e3d54.rlib /target/aarch64-unknown-linux-musl/debug/deps/libos_info-98fc2186f1d54fb8.rlib /target/aarch64-unknown-linux-musl/debug/deps/libpaketkoll_core-ae547def72c3c81d.rlib /target/aarch64-unknown-linux-musl/debug/deps/libglob-1ffa279201be60cf.rlib /target/aarch64-unknown-linux-musl/debug/deps/libbzip2-945f3f3f226a53e1.rlib /target/aarch64-unknown-linux-musl/debug/deps/libbzip2_sys-f7690df051249ee2.rlib /target/aarch64-unknown-linux-musl/debug/deps/libxz2-cd3cb3c69fb8d444.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblzma_sys-7ba82b51a8331d3d.rlib /target/aarch64-unknown-linux-musl/debug/deps/libflume-fdb1a33c515be94a.rlib /target/aarch64-unknown-linux-musl/debug/deps/libnum_cpus-f2b76db177a41b30.rlib /target/aarch64-unknown-linux-musl/debug/deps/libar-163ea80ebbacf220.rlib /target/aarch64-unknown-linux-musl/debug/deps/libtar-04148244a17ea86e.rlib /target/aarch64-unknown-linux-musl/debug/deps/libxattr-3f9ae327ef84a2da.rlib /target/aarch64-unknown-linux-musl/debug/deps/librustix-b0b9d34f3ef1e78f.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblinux_raw_sys-21b20448246edf68.rlib /target/aarch64-unknown-linux-musl/debug/deps/libfiletime-64bc801f6c017776.rlib /target/aarch64-unknown-linux-musl/debug/deps/libzstd-aec18906597eb4f0.rlib /target/aarch64-unknown-linux-musl/debug/deps/libzstd_safe-e2a63584664c1791.rlib /target/aarch64-unknown-linux-musl/debug/deps/libzstd_sys-32669feaf9b0441b.rlib /target/aarch64-unknown-linux-musl/debug/deps/libparking_lot-7cf960452f5a6b17.rlib /target/aarch64-unknown-linux-musl/debug/deps/libini-5696f8e086681921.rlib /target/aarch64-unknown-linux-musl/debug/deps/libtrim_in_place-15ef2ebe75ec2ccb.rlib /target/aarch64-unknown-linux-musl/debug/deps/libordered_multimap-f2cafd2bda2d6c6c.rlib /target/aarch64-unknown-linux-musl/debug/deps/libdlv_list-a95616a16286057c.rlib /target/aarch64-unknown-linux-musl/debug/deps/libphf-37f30b4232bccf14.rlib /target/aarch64-unknown-linux-musl/debug/deps/libphf_shared-3a4680b25332d1d8.rlib /target/aarch64-unknown-linux-musl/debug/deps/libsiphasher-d8e0f84d169cc7fc.rlib /target/aarch64-unknown-linux-musl/debug/deps/libignore-96c9f1a3677e5739.rlib /target/aarch64-unknown-linux-musl/debug/deps/libwalkdir-999bce26bf374208.rlib /target/aarch64-unknown-linux-musl/debug/deps/libsame_file-358a8f2db3e80589.rlib /target/aarch64-unknown-linux-musl/debug/deps/libglobset-c3c2301d06d0a237.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblog-2e501f34727b15ac.rlib /target/aarch64-unknown-linux-musl/debug/deps/libmd5-1b031c122055d5bd.rlib /target/aarch64-unknown-linux-musl/debug/deps/libdigest-eb9701d75aedabd9.rlib /target/aarch64-unknown-linux-musl/debug/deps/libblock_buffer-f959b7b325b2998d.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcrypto_common-136002bbd9c8e538.rlib /target/aarch64-unknown-linux-musl/debug/deps/libgeneric_array-fce974acd905d371.rlib /target/aarch64-unknown-linux-musl/debug/deps/libtypenum-982618236dc3952f.rlib /target/aarch64-unknown-linux-musl/debug/deps/libpaketkoll_utils-c0238f7f4ee8aa17.rlib /target/aarch64-unknown-linux-musl/debug/deps/libring-54baa33c753abe23.rlib /target/aarch64-unknown-linux-musl/debug/deps/libspin-95720c1a8683caf1.rlib /target/aarch64-unknown-linux-musl/debug/deps/libuntrusted-2c9fa63b699cec54.rlib /target/aarch64-unknown-linux-musl/debug/deps/libmtree2-1df96acda0a011a5.rlib /target/aarch64-unknown-linux-musl/debug/deps/libflate2-e57f32eeb579768a.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblibz_ng_sys-2df53b732c0ea07a.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcrc32fast-fa30ed19936b0fb9.rlib /target/aarch64-unknown-linux-musl/debug/deps/libregex-b1c38795bf4b10a6.rlib /target/aarch64-unknown-linux-musl/debug/deps/libbstr-885d1ea510e380d0.rlib /target/aarch64-unknown-linux-musl/debug/deps/libregex_automata-645483a204ad645a.rlib /target/aarch64-unknown-linux-musl/debug/deps/libaho_corasick-5447e2cf3fdc87dc.rlib /target/aarch64-unknown-linux-musl/debug/deps/libregex_syntax-49fe88d2665ab5bd.rlib /target/aarch64-unknown-linux-musl/debug/deps/libmemchr-2c1d667111a5d069.rlib /target/aarch64-unknown-linux-musl/debug/deps/libpaketkoll_types-07c7b66d6d55f314.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblasso-f5da76b4a5991cb2.rlib /target/aarch64-unknown-linux-musl/debug/deps/libdashmap-2a3b7ef5de893ca4.rlib /target/aarch64-unknown-linux-musl/debug/deps/libhashbrown-1218c9c558352c87.rlib /target/aarch64-unknown-linux-musl/debug/deps/libfaster_hex-a61728ceb202e328.rlib /target/aarch64-unknown-linux-musl/debug/deps/libserde_bytes-802fb2cbb58fefab.rlib /target/aarch64-unknown-linux-musl/debug/deps/libnix-394b1843d91b4cbb.rlib /target/aarch64-unknown-linux-musl/debug/deps/libderive_builder-ce8f3714191c2764.rlib /target/aarch64-unknown-linux-musl/debug/deps/libbitflags-fd4f6c46ab290a5f.rlib /target/aarch64-unknown-linux-musl/debug/deps/libthiserror-ca4c9ada60adf778.rlib /target/aarch64-unknown-linux-musl/debug/deps/libstrum-178ce37859f88706.rlib /target/aarch64-unknown-linux-musl/debug/deps/libdashmap-aee948a18804644c.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblock_api-f585f6dc36ad1b57.rlib /target/aarch64-unknown-linux-musl/debug/deps/libscopeguard-8bd0d056ec1684c0.rlib /target/aarch64-unknown-linux-musl/debug/deps/libhashbrown-37e22e4fc15549f8.rlib /target/aarch64-unknown-linux-musl/debug/deps/liballocator_api2-b679d84ff60160ae.rlib /target/aarch64-unknown-linux-musl/debug/deps/libparking_lot_core-a7181e9afaae8341.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcompact_str-6987da3bf8766e73.rlib /target/aarch64-unknown-linux-musl/debug/deps/libryu-e8bca8abb3d31978.rlib /target/aarch64-unknown-linux-musl/debug/deps/libitoa-1e607d164f4083d3.rlib /target/aarch64-unknown-linux-musl/debug/deps/libstatic_assertions-fc2c5dfa1d1a5dab.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcastaway-19be1111bd3c659f.rlib /target/aarch64-unknown-linux-musl/debug/deps/libsmallvec-7c37c71e4d75c193.rlib /target/aarch64-unknown-linux-musl/debug/deps/libserde-4be2fe20c57f2b9e.rlib /target/aarch64-unknown-linux-musl/debug/deps/libmimalloc-b7738520f972514e.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblibmimalloc_sys-6c0e9186df988277.rlib /target/aarch64-unknown-linux-musl/debug/deps/librayon-39c1a8c4f41afd61.rlib /target/aarch64-unknown-linux-musl/debug/deps/librayon_core-411c37910918dfbe.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcrossbeam_deque-958d910d3fc9c7f0.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcrossbeam_epoch-951dda309020f21e.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcrossbeam_utils-cc5589b8e8f15bbc.rlib /target/aarch64-unknown-linux-musl/debug/deps/libeither-4a3ae4cfdcf7bdb8.rlib /target/aarch64-unknown-linux-musl/debug/deps/libproc_exit-8498b1338578edba.rlib /target/aarch64-unknown-linux-musl/debug/deps/libclap-7366d26291cb12fd.rlib /target/aarch64-unknown-linux-musl/debug/deps/libclap_builder-c4c5e5dcb350e76a.rlib /target/aarch64-unknown-linux-musl/debug/deps/libstrsim-06b7fead5a98fecb.rlib /target/aarch64-unknown-linux-musl/debug/deps/libanstream-67872ffc57a29db7.rlib /target/aarch64-unknown-linux-musl/debug/deps/libanstyle_query-7780f943e8b484c0.rlib /target/aarch64-unknown-linux-musl/debug/deps/libis_terminal_polyfill-3c6fb2c6797f50c5.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcolorchoice-c94897b0d0c571bb.rlib /target/aarch64-unknown-linux-musl/debug/deps/libanstyle_parse-c0dc749435b4e727.rlib /target/aarch64-unknown-linux-musl/debug/deps/libutf8parse-bb165e27c62f6a5d.rlib /target/aarch64-unknown-linux-musl/debug/deps/libclap_lex-02eb4a5e56d4b271.rlib /target/aarch64-unknown-linux-musl/debug/deps/libanstyle-0c22d89388f1b183.rlib /target/aarch64-unknown-linux-musl/debug/deps/libanyhow-ace75f78c86bb2bf.rlib /target/aarch64-unknown-linux-musl/debug/deps/libahash-a13908ed97ec726c.rlib /target/aarch64-unknown-linux-musl/debug/deps/libgetrandom-1a57b3a48ba86154.rlib /target/aarch64-unknown-linux-musl/debug/deps/liblibc-57bc4bd4acb02d83.rlib /target/aarch64-unknown-linux-musl/debug/deps/libonce_cell-85a46c04a094204c.rlib /target/aarch64-unknown-linux-musl/debug/deps/libportable_atomic-89d5fc52adbbe488.rlib /target/aarch64-unknown-linux-musl/debug/deps/libcfg_if-dd139f6561fa45b0.rlib /target/aarch64-unknown-linux-musl/debug/deps/libzerocopy-dc998ed92317929b.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libstd-6e22b5d64cda649f.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libpanic_unwind-6dbdda2b1896f9f5.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libobject-7c15364b571acde3.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libmemchr-5f08222295da918c.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libaddr2line-b3271a0e792541e1.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libgimli-ddb92b1d3a7f3fa9.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/librustc_demangle-0f203aa5e4445b63.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libstd_detect-51bc41b0313f22d5.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libhashbrown-c6e9d546aea92afd.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/librustc_std_workspace_alloc-6a0e538de4ea440e.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libminiz_oxide-d08061ebdb7684ff.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libadler-3c4c44baabf4dded.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libunwind-b9f2df18bad75795.rlib -lunwind /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libcfg_if-3d127a20420d1d03.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/liblibc-f1d0ee652437bf5c.rlib -lc /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/liballoc-a2d865e3d00a5fb6.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/librustc_std_workspace_core-eb3ad2970efcad72.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libcore-2a67b8b32bd9e6f3.rlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/libcompiler_builtins-355b08253bc8d828.rlib -Wl,-Bdynamic -Wl,--eh-frame-hdr -Wl,-z,noexecstack -nostartfiles -L /rust/lib/rustlib/aarch64-unknown-linux-musl/lib -L /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained -o /target/aarch64-unknown-linux-musl/debug/deps/paketkoll-56d15b94d8d25e20 -Wl,--gc-sections -static -no-pie -Wl,-z,relro,-z,now -nodefaultlibs -Wl,--compress-debug-sections=zlib /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crtend.o /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/crtn.o
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/libc.a(vfprintf.lo): in function `pop_arg':
          /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:128: undefined reference to `__extenddftf2'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/libc.a(vfprintf.lo): in function `fmt_fp':
          /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:210: undefined reference to `__addtf3'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:228: undefined reference to `__subtf3'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:229: undefined reference to `__addtf3'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:244: undefined reference to `__fixtfsi'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:246: undefined reference to `__subtf3'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:246: undefined reference to `__multf3'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:225: undefined reference to `__multf3'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:232: undefined reference to `__addtf3'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:233: undefined reference to `__subtf3'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:268: undefined reference to `__multf3'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:274: undefined reference to `__fixunstfsi'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:275: undefined reference to `__subtf3'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:275: undefined reference to `__multf3'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/stdio/vfprintf.c:331: undefined reference to `__addtf3'
          /usr/local/bin/../lib/gcc/aarch64-linux-musl/9.2.0/../../../../aarch64-linux-musl/bin/ld: /rust/lib/rustlib/aarch64-unknown-linux-musl/lib/self-contained/libc.a(frexpl.lo): in function `frexpl':
          /build/musl-cross-make/build/local/aarch64-linux-musl/obj_musl/../src_musl/src/math/frexpl.c:16: undefined reference to `__multf3'
          collect2: error: ld returned 1 exit status
```

### Version it worked on

It most recently worked on: 1.82.0-nightly (2cbbe8b8b 2024-07-28) (or stable 1.80.0 or beta 1.81.0, take your pick for triaging)

### Version with regression

`rustc --version --verbose`:
```
rustc 1.82.0-nightly (612a33f20 2024-07-29)
binary: rustc
commit-hash: 612a33f20b9b2c27380edbc4b26a01433ed114bc
commit-date: 2024-07-29
host: x86_64-unknown-linux-gnu
release: 1.82.0-nightly
LLVM version: 18.1.7
```

--- a/library/alloc/Cargo.toml
+++ b/library/alloc/Cargo.toml
@@ -10,10 +10,7 @@ edition = "2021"
 
 [dependencies]
 core = { path = "../core" }
-compiler_builtins = { version = "0.1.114", features = ['rustc-dep-of-std'] }
-
-[target.'cfg(not(any(target_arch = "aarch64", target_arch = "x86", target_arch = "x86_64")))'.dependencies]
-compiler_builtins = { version = "0.1.114", features = ["no-f16-f128"] }
+compiler_builtins = { version = "0.1.117", features = ['rustc-dep-of-std'] }
 
 [dev-dependencies]
 rand = { version = "0.8.5", default-features = false, features = ["alloc"] }

--- a/library/std/Cargo.toml
+++ b/library/std/Cargo.toml
@@ -17,7 +17,7 @@ cfg-if = { version = "1.0", features = ['rustc-dep-of-std'] }
 panic_unwind = { path = "../panic_unwind", optional = true }
 panic_abort = { path = "../panic_abort" }
 core = { path = "../core", public = true }
-compiler_builtins = { version = "0.1.114" }
+compiler_builtins = { version = "0.1.117" }
 profiler_builtins = { path = "../profiler_builtins", optional = true }
 unwind = { path = "../unwind" }
 hashbrown = { version = "0.14", default-features = false, features = [

